Makefile

```diff
@@ -114,6 +114,16 @@ lib-release-test: lib-package
 lib-release: lib-package
 	twine upload dist/*
 
+lib-doc:
+	pushd docs && \
+	python -m sphinx \
```

Comment:
No need to maintain a separate copy of the build process. Call `tox` like everybody else in the ecosystem. The docs deps are conceptually different from the test ones. By mixing them up you increase the chance of hitting pointless dependency conflicts.
Just do `tox -e build-docs`.

Reply:
https://github.com/abhinavsingh/proxy.py/pull/752